app/controllers/member/dashboard_controller.rb

```diff
@@ -2,6 +2,8 @@ class Member::DashboardController < Member::BaseController
 
   def index
     authorize!(:show, :member_dashboard)
+    authorize!(:index, List)
```

Comment:
To be safe and consistent, in the future if you need to authorize against all lists you maybe you could just assign `@lists = Lists.all` then `authorize(:index, @lists)`